app/models/changeset/check_suite.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+class Changeset::CheckSuite
+  attr_reader :repo, :data
+
+  def initialize(repo, data)
+    @repo = repo
+    @data = data
+  end
+
+  def self.changeset_from_webhook(project, payload)
+    new(project.repository_path, payload)
+  end
+
+  def self.valid_webhook?(payload)
+    payload['check_suite']['status'] == 'completed' && payload['check_suite']['conclusion'] == 'success'
```

Comment:
if it has conclusion success, then it's also completed, right ?
```suggestion
    payload['check_suite']['conclusion'] == 'success'
```